.github/workflows/email-editor-package.yml

```diff
@@ -0,0 +1,82 @@
+name: Email Editor Package Tests
+
+on:
+  push:
+  pull_request:
+    branches:
+      - trunk
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-version: ['7.4', '8.2']
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Cache Composer vendor dependencies for MailPoet
+        id: composer-mailpoet-cache
+        uses: actions/cache@v4
+        with:
+          path: mailpoet/vendor
+          key: ${{ runner.os }}-composer-mailpoet-${{ matrix.php-version }}-${{ hashFiles('mailpoet/composer.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-composer-mailpoet-${{ matrix.php-version }}-
+            ${{ runner.os }}-composer-mailpoet-
+
+      - name: Cache Composer vendor-prefixed dependencies for MailPoet
+        id: vendor-prefixed-cache
+        uses: actions/cache@v4
+        with:
+          path: mailpoet/vendor-prefixed
+          key: ${{ runner.os }}-vendor-prefixed-${{ matrix.php-version }}-${{ hashFiles('mailpoet/prefixer/composer.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-prefixer-mailpoet-${{ matrix.php-version }}-
+            ${{ runner.os }}-prefixer-mailpoet-
```

Comment:
Why do we need to install these dependencies? If they are needed now, it's fine for this PR, but for the future, we should remove dependency on those or move them to monorepo root.  I'm asking basically to understand what needs to be done.

Reply:
There are more reasons for it.
- Codeception package, which is reused to avoid installing multiple times.
- Plugin activation in the tests docker entry point.
I will create a follow-up ticket for this.

Reply:
Got it! Thank you for explaining and for preparing the tickets 👍